lib/foods.js

```diff
@@ -1,6 +1,12 @@
+var foodList = JSON.parse(localStorage.getItem('foods')) || [];
 
 $('document').ready(function() {
 
+  foodList.forEach(function(element) {
```

Comment:
love this - i think you could wrap this all in a function though and just call it here 👍